### PR TITLE
potential fix to copy quizzes to manuscript dir

### PR DESCRIPTION
**Pull request by carriewright11**
#### What changes are being implemented in this Pull Request?

Modifying the bookdown_to_leanpub code to copy quizzes

#### What was your approach?

#79

### CODE REVIEW

File: R/bookdown_to_leanpub.R [hunk 1]
```diff
@@ -237,6 +244,10 @@ bookdown_to_leanpub <- function(path = ".",
     message("Copying bib files")
   }
   copy_bib(path, output_dir = output_dir)
+  copy_quizzes(path, output_dir = output_dir)
```
Comment (cansavvy) [inline R/bookdown_to_leanpub.R:247]: Just so its parallel to the others, can we put message first, then function?

**Review by cansavvy — APPROVED** [review]
It looks good to me (assuming checks pass) Ignore windows check still. UGh #73

File: R/bookdown_to_leanpub.R (hunk 1 above)
Comment (carriewright11) [reply R/bookdown_to_leanpub.R:247]: yup!

**Comment by carriewright11** [comment]
Look good now? we can make an issue about adding a check for quizzes later

File: R/bookdown_to_leanpub.R (hunk 1 above)
Comment (cansavvy) [reply R/bookdown_to_leanpub.R:251]: Yup that's it!

**Comment by cansavvy** [comment]
> Look good now? we can make an issue about adding a check for quizzes later

The checking machinery exists: https://jhudatascience.org/leanbuild/docs/reference/check_quizzes.html 
Just need to know what else I should add to it for checking: #52

**Pull request merged**